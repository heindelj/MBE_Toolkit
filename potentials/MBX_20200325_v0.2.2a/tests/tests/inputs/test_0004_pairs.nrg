SYSTEM H2O
MOLECULE
MONOMER h2o
 O                 -2.33449478   -0.87688733    0.00000000
 H                 -1.37449478   -0.87688733    0.00000000
 H                 -2.65494937    0.02804850    0.00000000
ENDMON h2o
ENDMOL
MOLECULE
MONOMER h2o
 O                  0.84785134    1.24854818    0.00000000
 H                  1.80785134    1.24854818    0.00000000
 H                  0.52739675    2.15348401    0.00000000
ENDMON h2o
ENDMOL
MOLECULE
MONOMER h2o
 O                  1.59117306   -1.77119626    0.00000000
 H                  2.55117306   -1.77119626    0.00000000
 H                  1.27071847   -0.86626043    0.00000000
ENDMON h2o
ENDMOL
MOLECULE
MONOMER h2o
 O                -57.46188630  -52.56027996    0.00000000
 H                -56.50188630  -52.56027996    0.00000000
 H                -57.78234089  -51.65534413    0.00000000
ENDMON h2o
ENDMOL
MOLECULE
MONOMER h2o
 O                -50.07177494  -52.56027996    0.00000000
 H                -49.11177494  -52.56027996    0.00000000
 H                -50.39222953  -51.65534413    0.00000000
ENDMON h2o
ENDMOL
MOLECULE
MONOMER h2o
 O                 67.26509525   56.17992621    0.00000000
 H                 68.22509525   56.17992621    0.00000000
 H                 66.94464067   57.08486204    0.00000000
ENDMON h2o
ENDMOL
ENDSYS
SYSTEM dummyn box of 100
MOLECULE
MONOMER dummy
 X                0.0 0.0 0.0
ENDMON dummy
ENDMOL
MOLECULE
MONOMER dummy
 X                1.0 0.0 0.0
ENDMON dummy
ENDMOL
MOLECULE
MONOMER dummy
 X                0.0 1.0 0.0
ENDMON dummy
ENDMOL
MOLECULE
MONOMER dummy
 X                0.0 0.0 1.0
ENDMON dummy
ENDMOL
MOLECULE
MONOMER dummy
 X               99.0 0.0 0.0
ENDMON dummy
ENDMOL
MOLECULE
MONOMER dummy
 X                0.0 99.0 0.0
ENDMON dummy
ENDMOL
MOLECULE
MONOMER dummy
 X                0.0 0.0 99.0
ENDMON dummy
ENDMOL
ENDSYS
