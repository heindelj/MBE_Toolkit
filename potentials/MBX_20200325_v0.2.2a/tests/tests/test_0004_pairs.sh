#!/bin/bash

filename=$(basename "$0")
filename="${filename%.*}"

mkdir -p outputs

../../install/bin/tests/combinations-test inputs/${filename}.nrg > outputs/${filename}.out

