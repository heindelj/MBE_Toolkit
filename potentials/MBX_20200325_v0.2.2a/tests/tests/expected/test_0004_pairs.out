SYSTEM 0
Dimer[         0] ->        0       1
Dimer[         1] ->        0       2
Dimer[         2] ->        1       2
Dimer[         3] ->        3       4
Trimer[         0] ->        0       1       2
SYSTEM 1 (7 mons)
Dimer[         0] ->        0       1
Dimer[         1] ->        0       2
Dimer[         2] ->        0       3
Dimer[         3] ->        0       4
Dimer[         4] ->        0       5
Dimer[         5] ->        0       6
Dimer[         6] ->        1       2
Dimer[         7] ->        1       3
Dimer[         8] ->        1       4
Dimer[         9] ->        1       5
Dimer[        10] ->        1       6
Dimer[        11] ->        2       3
Dimer[        12] ->        2       4
Dimer[        13] ->        2       5
Dimer[        14] ->        2       6
Dimer[        15] ->        3       4
Dimer[        16] ->        3       5
Dimer[        17] ->        3       6
Dimer[        18] ->        4       5
Dimer[        19] ->        4       6
Dimer[        20] ->        5       6
Trimer[         0] ->        0       1       2
Trimer[         1] ->        0       1       3
Trimer[         2] ->        0       1       4
Trimer[         3] ->        0       1       5
Trimer[         4] ->        0       1       6
Trimer[         5] ->        0       2       3
Trimer[         6] ->        0       2       4
Trimer[         7] ->        0       2       5
Trimer[         8] ->        0       2       6
Trimer[         9] ->        0       3       4
Trimer[        10] ->        0       3       5
Trimer[        11] ->        0       3       6
Trimer[        12] ->        0       4       5
Trimer[        13] ->        0       4       6
Trimer[        14] ->        0       5       6
Trimer[        15] ->        1       2       3
Trimer[        16] ->        1       2       4
Trimer[        17] ->        1       2       5
Trimer[        18] ->        1       2       6
Trimer[        19] ->        1       3       4
Trimer[        20] ->        1       3       5
Trimer[        21] ->        1       3       6
Trimer[        22] ->        1       4       5
Trimer[        23] ->        1       4       6
Trimer[        24] ->        1       5       6
Trimer[        25] ->        2       3       4
Trimer[        26] ->        2       3       5
Trimer[        27] ->        2       3       6
Trimer[        28] ->        2       4       5
Trimer[        29] ->        2       4       6
Trimer[        30] ->        2       5       6
Trimer[        31] ->        3       4       5
Trimer[        32] ->        3       4       6
Trimer[        33] ->        3       5       6
Trimer[        34] ->        4       5       6
